Anlab.Mvc/Controllers/LabController.cs

```diff
@@ -420,6 +421,43 @@ public IActionResult Orders(bool showComplete)
             return RedirectToAction("Details", new{id});
         }
 
+        public ActionResult Search()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> Search(string term)
+        {
+            Order order = null;
+            try
+            {
+                var shar = Guid.Parse(term);

```

Comment:
@jSylvestre I'd much rather you use the TryParse methods instead of catching exceptions for non-exceptional behavior.  I think you can probably do this w/o any exceptions either by using tryParse methods or by using a Regex to figure out if it looks like a guid, integer, or other.